tests/cpp/validator.cpp

```diff
@@ -106,30 +106,85 @@ void testValidate(
 
     if (aten_output_tensor.is_floating_point() ||
         aten_output_tensor.is_complex()) {
-      NVF_ERROR(
-          aten_output_tensor.allclose(
-              fusion_output_tensor.to(aten_output_tensor.dtype()),
-              tolerance_values.second,
-              tolerance_values.first,
-              /*equal_nan=*/true),
-          "\n",
-          err_msg,
-          "\nValidation error in output ",
-          i,
-          " on line ",
-          line_number,
-          " in file ",
-          file_name,
-          ".\n  Detected max abs error of: ",
-          aten_output_tensor.sub(fusion_output_tensor)
-              .abs()
-              .max()
-              .item()
-              .to<double>(),
-          "\n    absolute tolerance was set to ",
-          tolerance_values.first,
-          "\n    and relative tolerance set to ",
-          tolerance_values.second);
+      auto common_dtype = aten_output_tensor.dtype();
```

Comment:
Improved handling of fp8 in `testValidate`. Some aten ops for fp8 are missing, so we can not `at::allclose(fp8_t1, fp8_t2)`.